R/ecdc.R

```diff
@@ -59,19 +51,17 @@ get_ecdc_cases <- function (countries = NULL){
     
   }
 
-  d <- mem_read(base_url) %>%
+  data <- csv_reader(file = url) %>%
```

Comment:
I am a little confused - is the data read in twice (once on the error check and once here?)

Reply:
fixed so read in once only